src/openrct2-ui/input/ShortcutManager.h

```diff
@@ -61,12 +61,14 @@ namespace OpenRCT2::Ui
         std::vector<ShortcutInput> Default;
         std::vector<ShortcutInput> Current;
         std::function<void()> Action;
+        int16_t orderIndex = -1;
```

Comment:
I'm not certain as I'm new to this stuff, but I think if you'd put `static` in front of this line it should fix your linux compling issue

Reply:
@karst I thought orderindex was defined twice
I solved it through a new class constructor
thanks for the reply